tdrs-frontend/src/components/Header/Header.test.js

```diff
@@ -49,29 +85,93 @@ describe('Header', () => {
     expect(menuBtn).toExist()
   })
 
-  it("should add usa-current class to Welcome tab when on '/edit-profile'", () => {
-    const store = mockStore(initialState)
+  it("should add usa-current class to Welcome tab when on '/welcome'", () => {
+    const store = mockStore({
+      ...initialState,
+      router: { location: { pathname: '/welcome' } },
+    })
     const wrapper = mount(
       <Provider store={store}>
         <Header />
       </Provider>
     )
 
-    const welcomeTab = wrapper.find('.usa-nav__link')
+    const welcomeTab = wrapper.find('#welcome')
 
     expect(welcomeTab.hasClass('usa-current')).toEqual(true)
   })
 
-  it("should not add usa-current class to Welcome tab when not on '/edit-profile'", () => {
-    initialState = { ...initialState, router: { location: '/' } }
+  it("should add usa-current class to Reports tab when on '/reports'", () => {
+    const store = mockStore({
+      ...initialState,
+      router: { location: { pathname: '/reports' } },
+    })
+    const wrapper = mount(
+      <Provider store={store}>
+        <Header />
+      </Provider>
+    )
+
+    const reportsTab = wrapper.find('#reports')
+
+    expect(reportsTab.hasClass('usa-current')).toEqual(true)
+  })
+
+  it("should add usa-current class to Reports tab when on '/reports/*'", () => {
+    const store = mockStore({
+      ...initialState,
+      router: { location: { pathname: '/reports/upload' } },
+    })
+    const wrapper = mount(
+      <Provider store={store}>
+        <Header />
+      </Provider>
+    )
+
+    const reportsTab = wrapper.find('#reports')
+
+    expect(reportsTab.hasClass('usa-current')).toEqual(true)
+  })
+
+  it("should add usa-current class to Profile tab when on '/edit-profile'", () => {
```

Comment:
`Profile` should show active when on `/edit-profile`. This will be expanded once we add the actual profile page.